parts/classes/class-theme-customizer.php

```diff
@@ -270,6 +270,11 @@ public static function twentytwenty_get_color_options() {
 					'label'		=> __( 'Accent Color', 'twentytwenty' ),
 					'slug'		=> 'accent',
 				),
+				'twentytwenty_text_color' => array(
+					'default'	=> '#1A1B1F',
+					'label'		=> __( 'Text Color', 'twentytwenty' ),
+					'slug'		=> 'text',
```

Comment:
Again, use spaces for mid-line alignment, since tab width varies depending on the editor used and user preference.

Reply:
See https://github.com/WordPress/twentytwenty/pull/49#discussion_r322081817